fuzz/Cargo.toml

```diff
@@ -9,9 +9,6 @@ edition = "2021"
 [package.metadata]
 cargo-fuzz = true
 
-[dependencies]
-webpki = { package = "rustls-webpki", version = "0.101.0", features = ["alloc", "std"] }
```

Comment:
This is another change I'm having trouble matching up to the *ring* update. Could you split it off?

Reply:
(I still think this cleanup is unrelated to the ring update and would be better in a separate commit)

Reply:
taken the liberty of splitting this into #1523

Reply:
Thanks!